tests/import_test.go

```diff
@@ -2080,6 +2080,56 @@ var _ = Describe("Import populator", func() {
 		err = f.DeletePVC(pvcPrime)
 		Expect(err).ToNot(HaveOccurred())
 	})
+
+	It("should recreate and reimport pvc if it was deleted", func() {
+		dataVolume := utils.NewDataVolumeWithHTTPImport("import-dv", "100Mi", fmt.Sprintf(utils.TinyCoreIsoURL, f.CdiInstallNs))
+		dv, err := utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, dataVolume)
+		Expect(err).ToNot(HaveOccurred())
+
+		pvc, err = utils.WaitForPVC(f.K8sClient, dv.Namespace, dv.Name)
+		Expect(err).ToNot(HaveOccurred())
+		f.ForceBindIfWaitForFirstConsumer(pvc)
+
+		By("Verify PVC prime was created")
+		pvcPrime, err = utils.WaitForPVC(f.K8sClient, pvc.Namespace, populators.PVCPrimeName(pvc))
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
Just a dumb question (I guess this is done in many tests), can't PVC prime already be deleted in this point if import was quick enough to complete?

Reply:
yeah I agree, I did several changes in this test, since I found the deletePVCByUID then its safe enough to see that the new created PVC became bound again.. (I had a race there with the get). Ill simplify the test more.